build.sbt

```diff
@@ -127,7 +127,7 @@ lazy val deployedAppMappings = Seq(
 
   // The only thing we include from the base deployment app is app.conf. We remove the "conf" path.
   Compile / packageBin / mappings ~= {
-    _.filter(_._1.getName.endsWith(".conf")).map(mapping => mapping._1 -> mapping._1.getName)
+    _.filter(_._1.getName.endsWith(".conf.json")).map(mapping => mapping._1 -> mapping._1.getName)
```

Comment:
Why this?

Reply:
Because it wasn't including `environments.conf.json` which is what we need

Reply:
This is a common mapping for all 4 deployments. Do we want a common `environments.conf.json` for all 4 deployments?